server/routes/overview/overviewController.ts

```diff
@@ -1,35 +1,22 @@
 import { RequestHandler } from 'express'
-import type { Prisoner } from 'prisonRegisterApiClient'
-import createError from 'http-errors'
-import type { PrisonerSummary } from 'viewModels'
 import OverviewView from './overviewView'
-import PrisonerSearchService from '../../services/prisonerSearchService'
 
 export default class OverviewController {
-  constructor(private readonly prisonerSearchService: PrisonerSearchService) {}
-
   getOverviewView: RequestHandler = async (req, res, next): Promise<void> => {
```

Comment:
The implementation of `OverviewController` and it's test above are now much simpler 👍